server/models/dataset.py

```diff
@@ -28,6 +28,7 @@
 from girder.models.notification import ProgressState
 from girder.utility import assetstore_utilities, mail_utils
 from girder.utility.progress import ProgressContext
+from natsort import natsorted
```

Comment:
Since this is a third-party package dependency, can you put it in its own group between the builtin module imports and the Girder module imports?